packages/xrpl/src/Wallet/defaultFaucets.ts

```diff
@@ -1,4 +1,4 @@
-import type { Client } from '..'
+import type { Client } from '../client'
```

Comment:
Why this change?

Reply:
I don't know but I kind of like it better as it's more explicit from the source. Probably was getting an error in tests.